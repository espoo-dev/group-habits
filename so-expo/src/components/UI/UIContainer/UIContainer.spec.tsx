import { render, screen } from '@testing-library/react-native';
import { Text } from 'react-native';
import { UIContainer } from './index';

function setup() {
  return render(<UIContainer>
    <Text>
      Content Here
    </Text>
  </UIContainer>)
}

describe('src > components > UI > UIContainer', () => {
  it(`match snapshot`, () => {
    const tree = setup().toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`renders correctly`, () => {
    setup()

    const text = screen.getByText(/content here/i)

    expect(text).toBeOnTheScreen()
  })
});
