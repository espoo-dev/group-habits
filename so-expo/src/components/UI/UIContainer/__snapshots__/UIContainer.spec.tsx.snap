// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src > components > UI > UIContainer match snapshot 1`] = `
<View
  style={
    [
      {
        "display": "flex",
      },
      {
        "justifyContent": "space-between",
      },
      {
        "height": "100%",
      },
      {
        "width": "100%",
      },
      {
        "paddingLeft": 24,
        "paddingRight": 24,
      },
      {
        "paddingBottom": 96,
        "paddingTop": 96,
      },
    ]
  }
>
  <Text>
    Content Here
  </Text>
</View>
`;
